describealign.py

```diff
@@ -1109,8 +1109,27 @@ def combine_gui(video_files, audio_files, config_path):
       break
   combine_window.close()
 
+def migrate_config(old_path: Path, new_path: Path) -> None:
+  """
+  Migrate configuration from old location.
+  
+  Only runs if the old_path exists but new_path does not
+  """
+  if new_path.exists() or not old_path.exists():
+    return
+  
+  old_data = old_path.read_text(encoding='utf-8')
+  new_path.write_text(old_data, encoding='utf-8')
+  print(f"Configuration migrated to {new_path}")
```

Comment:
My concern is that the old config might not be writable, only readable to the current user (should be rare), but since I added the error check, I can add a delete yeah, should be fine.